showcase/README.md

```diff
@@ -0,0 +1,49 @@
+# Showcase integration testing
+
+This submodule contains the gapic-showcase integration tests for
+gapic-generator-go, as well as the script used to setup and execute them.
+
+## How the tests work
+
+The tests can be run out-of-the-box in the normal `go test` fashion, so long as
+there is a `gapic-showcase` server already running locally. This does not test
+the a client generated with the locally checked out generator - it would use the
+released version of the GAPIC. Running `make test` will execute the tests
+against the locally install generator.
+
+To test the local generator, the `showcase.bash` script first downloads the
```

Comment:
I would restructure this and the following paragraphs to be more list-like and active voice. WDYT?

```
To test the local generator, use the showcase.bash script. It does the following:
1. It downloads the ....
2. It generates the ... using protoc and ....
  - make sure to make install first....
3. It edits this submodule's go.mod....
4. It runs the server in the background
5. It runs the tests against the server
6. It stops the server once the tests are finished.
```

Reply:
Good idea! Done.